nbproject/customs.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
use gitignore to stop this folder nbproject from being tracked by git so it wont appear in your next commit. I wont be able to merge it cause its IDE specifc